playwright/e2e/crypto/utils.ts

```diff
@@ -22,6 +23,47 @@ import { Client } from "../../pages/client";
 import { ElementAppPage } from "../../pages/ElementAppPage";
 import { Bot } from "../../pages/bot";
 
+/**
+ * Create a bot and wait for it to be ready to use.
```

Comment:
Could you explain what a "bot" is in this context?

Reply:
I added a more explicit comment

Reply:
Thanks, at some point we can add some docs the the bot class itself, but that doesn't need to be this PR.